modules/psap-driver-toolkit-pulling.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/psap-driver-toolkit.adoc
+
+[id="pulling-the-driver-toolkit"]
+= Pulling the Driver Toolkit container image
+
+The driver-toolkit image is available on registry.redhat.io and in the OpenShift release payload. The image corresponding to the most recent minor release of {product-title} will be tagged with the version number on registry.redhat.io. The image URL for a specific release can be found using the `oc adm` CLI command.
+
+[id="pulling-the-driver-toolkit-from-registry"]
+== Pulling the Driver Toolkit container image from registry.redhat.io
+
+Instructions for pulling the driver-toolkit image from registry.redhat.io with podman, or in OpenShift can be found on the link:https://catalog.redhat.com/software/containers/openshift4/driver-toolkit-rhel8/604009d6122bd89307e00865?container-tabs=gti[Red Hat Ecosystem Catalog]. 
+The driver-toolkit image for the latest minor release will be tagged with the minor release version on registry.redhat.io for example `registry.redhat.io/openshift4/driver-toolkit-rhel8:v4.8`. 
+
+[id="pulling-the-driver-toolkit-from-payload"]
+== Finding the Driver Toolkit image URL in the payload
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+The image URL of the driver-toolkit corresponding to a certain release can be extracted from the release image using the `oc adm` command:
+
+[source,terminal]
+----
+$ oc adm release info 4.8.0 --image-for=driver-toolkit
```

Comment:
[source,terminal]
    ----
    $ oc adm release info 4.8.0 --image-for=driver-toolkit
    ----
    +
     .Example output
    [source,terminal]
    ----
    quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:&lt;SHA&gt;
    -----